### PR TITLE
CASMINST-3806 - add management network steps for adding mountain cabinet

### PR DESCRIPTION
**Pull request by dle-hpe**
# Description

added general guidelines for the networking portion of adding a liquid cooled mountain cabinet
# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Add_additional_Liquid-Cooled_Cabinets_to_a_System.md [hunk 1]
```diff
@@ -3,26 +3,49 @@
...
+1. Validate BIOS and BMC firmware levels in the new nodes.
+
+    Perform the procedures in [Update Firmware with FAS](../firmware/Update_Firmware_with_FAS.md). Perform updates as needed with FAS.
+
+    > Slingshot switches are updated with procedures from the *Slingshot Operations Guide*.
 
-6.  Continue on to the *Slingshot Operations Guide* to bring up the additional cabinets.
+1. Continue on to the *Slingshot Operations Guide* to bring up the additional cabinets.
 
     **NOTE:** Placeholder: Provide a link to the procedure once ready.
```
Comment (mharding-hpe) [inline operations/node_management/Add_additional_Liquid-Cooled_Cabinets_to_a_System.md:51]:
```suggestion
```

File: operations/node_management/Add_additional_Liquid-Cooled_Cabinets_to_a_System.md [hunk 2]
```diff
@@ -3,26 +3,48 @@
...
-5.  Perform procedures in [Update Firmware with FAS](../firmware/Update_Firmware_with_FAS.md) to validate BIOS and BMC firmware levels in the new nodes, and perform updates as needed with FAS.
-    > Slingshot Switches are updated with procedures from the *Slingshot Operations Guide*.
+1. Validate BIOS and BMC firmware levels in the new nodes.
+
+    Perform the procedures in [Update Firmware with FAS](../firmware/Update_Firmware_with_FAS.md). Perform updates as needed with FAS.
+
+    > Slingshot switches are updated with procedures from the *Slingshot Operations Guide*.
 
-6.  Continue on to the *Slingshot Operations Guide* to bring up the additional cabinets.
+1. Continue on to the *Slingshot Operations Guide* to bring up the additional cabinets.
```
Comment (mharding-hpe) [inline operations/node_management/Add_additional_Liquid-Cooled_Cabinets_to_a_System.md:50]:
```suggestion
```

**Review by mharding-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
Needs a manual main backport.

**Comment by mharding-hpe** [comment]
> Needs a manual main backport.

Created manual backport: https://github.com/Cray-HPE/docs-csm/pull/1917

**Pull request merged**